apps/src/code-studio/pd/foorm/editor/form/FoormFormSaveBar.jsx

```diff
@@ -392,6 +415,16 @@ const styles = {
   }
 };
 
+const saveNewVersionWarning = version => (
+  <div>
+    <span style={styles.warning}>Warning: </span>You are about to save version{' '}
+    {version} of this form. This version will need to be deployed separately.
```

Comment:
Not sure a content editor will understand this sentence? Guessing you mean in the context of a Simple Survey you need to re-specify the version of the form that will appear on a page.

Maybe a reference to the "When to Version" section of this wiki:

https://github.com/code-dot-org/code-dot-org/wiki/%5BLevelbuilder%5d-The-Foorm-Editor#when-to-version

And a note to check with an engineer if unsure?